libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -925,6 +925,20 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 						"(" <<
 						IRVariable(arg).commaSeparatedList() <<
 						")";
+				else if (auto functionType = dynamic_cast<FunctionType const*>(paramTypes[i]))
+				{
+					solAssert(
+						IRVariable(arg).type() == *functionType &&
+						functionType->kind() == FunctionType::Kind::External &&
+						!functionType->bound(),
+						""
+					);
+					define(indexedArgs.emplace_back(m_context.newYulVariable(), *TypeProvider::uint256())) <<
```

Comment:
Shouldn't it be `bytes24`?

Reply:
Why? `combineExternalFunctionIdFunction()` adds padding.

Reply:
it's 20 bytes of address followed by 4 bytes of function selector, followed by 8 zero bytes. It's certainly not an unsigned integer.

Reply:
So would `fixedBytes(32)` be more appropriate? Would there be a difference between using `uint256` and `fixedBytes(32)`?

Reply:
There should not be a real difference in this context, I just find it more appropriate, yes.

Reply:
Ok I changed it to `fixedBytes(32)`.